specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/storage.json

```diff
@@ -4741,7 +4741,7 @@
           "minimum": 0,
           "description": "Value indicating the age in days after creation"
         },
-        "daysAfterLastTierChangeTimeGreaterThan": {
+        "daysAfterLastTierChangeGreaterThan": {
```

Comment:
the "DateAfterCreation"description on line 4754 looks is not right now ("Object to define the number of days after creation."), since "daysAfterLastTierChangeGreaterThan" added.

Reply:
I have updated the description